resources/views/consultants/show.blade.php

```diff
@@ -22,29 +22,36 @@
             @csrf
             @method('PUT')
 
-            <x-hearth-input type="submit" name="unpublish" :value="__('consultant.action_unpublish')" />
+            <x-hearth-input type="submit" name="unpublish" :value="__('Unpublish my page')" />
         </form>
         @endif
         @endcan
     </x-slot>
 
 
-    <div class="tabs flow" x-data="tabs(window.location.hash ? window.location.hash.substring(1) : 'about')" x-on:resize.window="enabled = window.innerWidth > 1023">
-        <h2 x-show="!enabled">{{ __('Contents') }}</h2>
-        <ul x-bind="tabList">
-            <li x-bind="tabWrapper"><a href="#about" x-bind="tab">{{ __('consultant.section_about') }}</a></li>
-            <li x-bind="tabWrapper"><a href="#interests-and-goals" x-bind="tab">{{ __('consultant.section_interests_and_goals') }}</a></li>
-            @can('update', $consultant)<li x-bind="tabWrapper"><a href="#lived-experience" x-bind="tab">{{ __('consultant.section_lived_experience') }}</a></li>@endcan {{--  TODO: handle this more granularly --}}
-            <li x-bind="tabWrapper"><a href="#professional-experience" x-bind="tab">{{ __('consultant.section_professional_experience') }}</a></li>
-            @can('update', $consultant)<li x-bind="tabWrapper"><a href="#access-needs" x-bind="tab">{{ __('consultant.section_access_needs') }}</a></li>@endcan {{--  TODO: handle this more granularly --}}
-        </ul>
-        <div class="flow" id="about" x-bind="tabpanel">
-            <h2>{{ __('consultant.section_about_person', ['name' => $consultant->firstName()]) }}</h2>
+    <div class="has-nav-secondary">
+        <nav class="secondary" aria-labelledby="project">
```

Comment:
The `projects` id doesn't exist on an element in the consultants page.

Reply:
Resolved in effdf6b.